docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,191 @@
+{
+  prometheusAlerts+:: {
```

Comment:
I agree, it would be better to have them more closely, especially for reviewing. Sadly this is the current way of defining alerts and rules in the mixin spec.

Reply:
@metalmatze Is that really so? If I simply added the current alert rules to the `prometheusRules.groups[0]`, would something break?

I mean, it's not just a question of nicer reviews. It's actually quite important to have alerts depending on recording rules in the same group so that they get evaluated just after the recording rule.

I'll play with it and see if it works.

Reply:
Wait, none of our alerts depends on a recording rule.
I think this is fine for now, but we should keep in mind to put recording rules needed for alerts into the same group, and if that's not possible, make it possible within the mixin spec.

Reply:
I think there is nothing to do in the scope of this PR.